components/ai_chat/resources/page/state/ai_chat_context.tsx

```diff
@@ -142,9 +78,16 @@ export function AIChatContextProvider(props: React.PropsWithChildren<Props>) {
   const store: AIChatContext = {
     ...context,
     ...props,
-    isStandalone: getAPI().isStandalone,
     goPremium: () => UIHandler.goPremium(),
     managePremium: () => UIHandler.managePremium(),
+    markStorageNoticeViewed: () => Service.dismissStorageNotice(),
+    dismissStorageNotice: () => {
+      setPartialContext({
```

Comment:
just checking, but this is just eagerly changing the data right? After the "backend" does the update we'll receive a notification anyway